packages/sitecore-jss/src/utils/env.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Parses raw env variable value into a Record
+ * @param {string} mvEnv input
+ * @returns non-empty Record
+ * @throws error when input JSON can't be parsed
+ */
+export const parseMultiValueAsRecord = <T>(mvEnv: string): T => {
```

Comment:
We shouldn't have anything referencing a "Record" any longer. Do we even need this function? Seems like this can just be rolled into `tryParseEnvValue` now.